src/index.js

```diff
@@ -185,6 +185,22 @@ function getMatchingElements (elementIterator, ast, multiple) {
   return results
 }
 
+function getMatchingElementsByClassName (elementIterator, classNames) {
+  const results = [];
+  let element
+
+  while ((element = elementIterator.next())) {
+    const elementClassList = element.classList;
+    const contains = classNames.every(function(className) {
+      return [].slice.call(elementClassList).indexOf(className) >= 0;
+    });
```

Comment:
Why not just `element.classList.contains()`?

Reply:
Great catch, I confused `contains` with `includes`